Editing FAQ. (#17153)
metabase/metabase: docs/faq/using-metabase/how-do-i-answer-questions-when-data-is-in-multiple-databases.md

# How do I answer questions about data that lives in multiple databases?

> Metabase does not currently support joining tables across multiple databases. It is generally better to bring related data into a single database.

Reviewer: ```suggestion
Metabase doesn't currently support joining tables across multiple databases. It's generally better to bring related data into a single database.
```
